kie-server-parent/kie-server-controller/kie-server-controller-websocket/src/test/java/org/kie/server/controller/websocket/management/KieServerMgmtCommandServiceImplTest.java

```diff
@@ -195,4 +200,28 @@ public void testUpdateContainerConfig() {
         assertEquals("No server template found for id serverTemplateId", response.getMsg());
         assertNull(response.getResult());
     }
+
+    @Test
+    public void testStartScanner() {
+        final ContainerSpecKey containerSpecKey = new ContainerSpecKey("id",
+                                                                       "name",
+                                                                       new ServerTemplateKey("serverTemplateId",
+                                                                                             "stname"));
+        KieServerControllerDescriptorCommand command = new KieServerControllerDescriptorCommand(RuleCapabilitiesService.class.getName(),
+                                                                                                "startScanner",
+                                                                                                null,
+                                                                                                null,
+                                                                                                containerSpecKey,
+                                                                                                new Long(1));
+        final String content = WebSocketUtils.marshal(command);
+        LOGGER.info("JSON content\n{}", content);
```

Comment:
maybe change to debug to reduce log size...

Reply:
Done